service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrganisationService.java

```diff
@@ -56,4 +58,14 @@
             .map(User::getUserIdentifier)
             .collect(toList());
     }
+
+    public Optional<String> findUserByEmail(String authorisation, String email) {
+        try {
+            return Optional.of(organisationApi.findUserByEmail(authorisation, authTokenGenerator.generate(), email)
+                .getUserIdentifier());
+        } catch (FeignException.NotFound notFoundException) {
+            log.warn("User with email {} not found", email);
```

Comment:
Where does that email come from? Is it going to happen frequently that we don't find user with given email? Because it looks to me that this should be logged on `debug` level.

Reply:
done